Hospital/src/patterns/RoomFactory.java

```diff
@@ -0,0 +1,17 @@
+package patterns;
+
+public class RoomFactory {
+	
+	public Room getRoom(String tipo){
+		
+		if (tipo.equals("cuarto comun"))
+			return new HabitacionComun();
+		if (tipo.equals("quirofano"))
+			return new Quirofano();
+		if (tipo.equals("ba�o"))
+			return new Ba�o();
+
+		return null;
+	}
+
+}
```

Comment:
[General Comments]
Please set the formatter file in your project. If you are using eclipse, you have to:
1. Go to project -> properties
2. Select Java Code Style -> Formatter
3. Enable project specific settings
4. Click on Import...
5. Select formatter.xml provided by bootcamp course
6. Click ok

Then, you can format each file manually (you have the option in Source menu) or you can do it automatically when saving the file. To enable this option you have to do the following:
1. Go to project -> properties
2. Select Java Editor -> Save actions
3. Enable option Enable project specific settings
4. Enable option Perform the selected actions on save
5. Enable option Format source code
6. Select option format all lines

[General Comment]
Start including javadoc in most important classes, methods and properties.